NORSE_actor/externalNORSERun.py

```diff
@@ -270,4 +290,22 @@
 
 # put CPO
 itmp.distributionArray.put()
-itmp.close()
\ No newline at end of file
+itmp.close()
+
+# Reshape data for hdf5 writing
+temperature = temperature.reshape(1,rho_size)
+density = density.reshape(1,rho_size)
+EHat = EHat.reshape(1,rho_size)
+Z_eff = Z_eff.reshape(1,rho_size)
+B0 = B0.reshape(1,rho_size)
+rhoTor = rhoTor_arr.reshape(1,rho_size)
+E_parallel = E_parallel.reshape(1,rho_size)
+E_critical = E_critical.reshape(1,rho_size)
+time = time.reshape(1,1)
+
+growth_rate = np.array(growth_rate).reshape(1,rho_size)
+runaway_density = np.array(runaway_density).reshape(1,rho_size)
+runaway_current = np.array(runaway_current).reshape(1,rho_size)
+
+# Write data to hdf5 file
+hdf5Write.write(shot, run, temperature.reshape(1,rho_size), get_name(temperature), density, get_name(density), rhoTor, get_name(rhoTor), B0, get_name(B0), Z_eff, get_name(Z_eff), EHat, get_name(EHat), E_parallel, get_name(E_parallel), E_critical, get_name(E_critical), time, get_name(time), growth_rate, get_name(growth_rate), runaway_density, get_name(runaway_density), runaway_current, get_name(runaway_current))
```

Comment:
distribution and all important outputs should also be saved. The aim is to have a full set of outputs in HDF5.

Reply:
Should be done similarly as for the others, but for each rho coordinate there will be a vector (Bigvector in NORSE naming) of distribution and the coordinates instead of a single number like for the other parameters. I will have to find a way to do this.

Reply:
Done.